src/Hydra/JsonSchema/SchemaFactory.php

```diff
@@ -121,7 +121,10 @@ public function buildSchema(string $className, string $format = 'jsonld', string
                                 'properties' => [
                                     '@type' => ['type' => 'string'],
                                     'variable' => ['type' => 'string'],
-                                    'property' => ['type' => 'string'],
+                                    'property' => [
+                                        'nullable' => true,
```

Comment:
`'nullable' => true` is only for OpenAPI < 3.1.
I don't think we should add it here.

Reply:
Ok. the change from string to `'type' => ['string', 'null'],` broke Behat pipelien, I'll update the tests as well if you're OK with the change ?

Reply:
Yes I think this change is necessary.

Reply:
It seems that `'null'` is [valid type starting on Open-API 3.1](https://stackoverflow.com/questions/48111459/how-to-define-a-property-that-can-be-string-or-null-in-openapi-swagger#answer-48114322) and the schema generated by the `api:openapi:export` command is at "3.0.2", so it wont pass swagger-cli validation (which is normal). I could check the version to add `nullable` or `['string', 'null']` combo, but I don't think you have control over the generated minor version. Any idea on how to solve this ?

Reply:
It's more or less what's has been done here: https://github.com/api-platform/core/blob/f53bb64825950bf1da2b91e12673b465d3d5312b/src/JsonSchema/TypeFactory.php#L155-L184

Reply:
Fixed ! I've kept nullable for `openapi` version, but if it's not necessary I'll move it to a ternary.